src/Rendering/babylon.geometryBufferRenderer.ts

```diff
@@ -2,13 +2,12 @@ module BABYLON {
     export class GeometryBufferRenderer {
         private _scene: Scene;
         private _multiRenderTarget: MultiRenderTarget;
-        private _effect: Effect;
         private _ratio: number;
-
-        private _cachedDefines: string;
-
         private _enablePosition: boolean = false;
 
+        protected _effect: Effect;

```

Comment:
final question - do you need to change and set this one? or will a getter be enough?

Reply:
Sorry, I looked at the class, it makes sense, since you want to set them in the function. All good.